### ISSUE TITLE
Collections in the WWT web client

### ISSUE THREAD

**Issue by erikasilvads** [WorldWideTelescope/wwt-web-client#357]
@astrodavid10 and I noticed that we aren't able to create collections from within the WWT web client, and that would be great if that functionality was implemented.

**Comment by patudom** [comment]
Yes, agree that would be extremely helpful functionality to add to the web client!

**Comment by pkgw** [comment]
Just to clarify what you mean ... Are you thinking, like, interfaces so that you can create a new folder, put images etc. into it, then download a resulting WTML file?

**Comment by patudom** [comment]
Yes. I don't know about @erikasilvads  and @astrodavid10's use case, but what you describe, Peter, is how we generate WTML files that we need (which currently only works in the desktop client). It would be great if we could do this in the web client, so I don't have to dust off my Windows computer every time we need a WTML file.

Our group would also be happy with an easy way to convert a WCS-tagged image (as from AstroPix) directly into a WTML file. @johnarban hacked a way to do this for our Green Comet Mini Data Story, but a more straightforward pathway would be great. (John, feel free to explain further if I don't quite have that correct).

Thanks!

**Comment by erikasilvads** [comment]
Yes! That's what I meant.

**Comment by pkgw** [comment]
@patudom For the second thing you mention, the `toasty` Python package does provides that functionality, via the [`toasty tile-study`][1] command line program. Unfortunately the relevant documentation only gives the basic reference information — it really needs some tutorials and examples. Next time you need to do something like that, let me know and I'll show you what it can do and we can see if it meets your needs.

[1]: https://toasty.readthedocs.io/en/latest/cli/tile-study.html#cli-tile-study

**Comment by johnarban** [comment]
so yeah, @astrodavid10 showed us that. And I think that is probably a better way than what I wrote, which just transforms a WCS header into the appropriate WTML tags (this works for degree/several arcminute scale images, but really small images will fail to render). It doesn't group data into collections or folders (which is not so hard) but the wwtdatatool will do that via  `wwtdatatool wtml merge ` I believe.
